server/service/src/backend_plugin/types/mod.rs

```diff
@@ -1,38 +1,3 @@
 pub mod amc;
 pub mod graphql_query;
 pub mod transform_request_requisition_lines;
```

Comment:
@GeronimoJohn, I think the types should live closer together.

I would make export_plugin_typescript() `pub` and use it in cli (less code changes too)